orttraining/orttraining/python/training/ortmodule/experimental/json_config/_load_config_from_json.py

```diff
@@ -246,6 +258,7 @@ def load_from_json(ortmodule, path=None):
             "FALLBACK_UNSUPPORTED_ONNX_MODEL",
             "FALLBACK_BAD_INITIALIZATION",
         ],
+        "OnnxOpsetVersion": 14 # int defining the opset version to be used during export
```

Comment:
```
from onnxruntime.training.ortmodule import ONNX_OPSET_VERSION`
```
https://github.com/microsoft/onnxruntime/blob/master/orttraining/orttraining/python/training/ortmodule/__init__.py#L40

Reply:
The changes here is for sake of completeness since we want to continue supporting the experimental json format. I know users can use the simple `from onnxruntime.training.ortmodule import ONNX_OPSET_VERSION`. This just extends the usability to json config.

Reply:
I meant that as opposed to hard code 14 as the default on line 261, you should use our global constant for it. If user specify a different value on the json, then it can be used as proposed :)

Reply:
yeah, I agree.
Actually, the code snippet that this comment is referring to is a docstring and it just for demonstrative purpose for users to understand how to use this. It does not define a default or the onnx opset version.
The onnx opset version is unchanged if it is not defined in the json file. So, this file does not redefine the ONNX opset constant.